federation/pkg/federatedtypes/hpa_test.go

```diff
@@ -216,8 +219,8 @@ func updateHpaStatus(hpa *autoscalingv1.HorizontalPodAutoscaler, currentUtilisat
 	now := metav1.Now()
 	scaledTime := now
 	if scaleable {
-		// definitely more then 5 minutes ago
-		scaledTime = metav1.NewTime(now.Time.Add(-6 * time.Minute))
+		// definitely more then "ScaleForbiddneWindow" (= 2 * time.Minute) ago
+		scaledTime = metav1.NewTime(now.Time.Add(-3 * time.Minute))
```

Comment:
Consider basing scaledTime on ScaleForbiddenWindow so that any change in that value does not require a change in this test.